desktop/config/README.md

```
@@ -0,0 +1,3 @@
+# Default Configurations
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)